spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/PubSubAdmin.java

```diff
@@ -95,6 +98,28 @@ public Topic createTopic(String topicName) {
 	}
 
 	/**
+	 * Get the configuration of a Google Cloud Pub/Sub topic.
+	 *
+	 * @param topicName canonical topic name, e.g., "topicName"
+	 * @return topic configuration or {@code null} if topic doesn't exist
+	 */
+	public Topic getTopic(String topicName) {
+		Assert.hasText(topicName, "No topic name was specified.");
+
+		try {
+			return this.topicAdminClient.getTopic(TopicName.create(this.projectId, topicName));
+		}
+		catch (ApiException aex) {
+			if (aex.getStatusCode() instanceof GrpcStatusCode
```

Comment:
gax version 1.11.0 allows us to deal with this more gracefully, but we need to wait for a google-cloud-pubsub release that uses that gax version.

Reply:
I have just updated https://github.com/kiwigrid/spring-cloud-stream-binder-google-pubsub so that `google-cloud-pub:0.26.0-beta` is also used there. Even with this version all tests are still green! So there's a mistake somewhere in this project here. I've already done some research, but I haven't been able to find the problem yet.